topics/sse_client.md

```diff
@@ -0,0 +1,90 @@
+[//]: # (title: Server-Sent Events)
+
+<show-structure for="chapter" depth="2"/>
+
+<tldr>
+<var name="example_name" value="client-sse"/>
+<include from="lib.topic" element-id="download_example"/>
+</tldr>
+
+<link-summary>
+The SSE plugin allows a client to receive event-based updates from a server over an HTTP connection.
+</link-summary>
+
+<include from="sse_server.md" element-id="sse-description" />
+
+
+> To learn more about the SSE plugin for server-side support, see [](sse_server.md).
+
+## Add dependencies {id="add_dependencies"}
+`SSE` only requires the [ktor-client-core](client-dependencies.md) artifact and doesn't need any specific dependencies.
+
+
+## Install SSE {id="install_plugin"}
+To install the `SSE` plugin, pass it to the `install` function inside a [client configuration block](create-client.md#configure-client):
+
+```kotlin
+import io.ktor.client.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.plugins.sse.*
+
+//...
+val client = HttpClient(CIO) {
+    install(SSE)
+}
+```
+
+## Configure the SSE plugin {id="configure"}
+
+[//]: # (TODO: Add link for SSEConfig)
+Optionally, you can configure the plugin inside the `install` block by passing the supported properties of the [SSEConfig]() class.
+* Use the `reconnectionTime` property to specify a reconnection time.  If the connection to the server is lost, the client will wait for the specified time before attempting to reconnect.
+* Use the `showCommentEvents()` function to add events consisting only of comments in the incoming flow.
+* Use the `showRetryEvents()` function to add events consisting only of the retry field in the incoming flow.
+
+```kotlin
+```
+{src="snippets/client-sse/src/main/kotlin/com.example/Application.kt" include-lines="11-15"}
+
+> Note that the `reconnectionTime` property is not supported in all engines.
```

Comment:
Can we place this note right under `reconnectionTime`? And maybe it's better to note that right now none of the engines support it?